kyaml/fn/runtime/starlark/example_test.go

```diff
@@ -131,7 +131,7 @@ spec:
   value: "hello world"
 `)
 	if err != nil {
-		log.Fatal(err)
+		log.Println(err)
```

Comment:
Why are we changing this?

Reply:
Fatal does os.Exit which will not perform defer cleanup. Lint caught that.